modules/ROOT/pages/composite-databases/index.adoc

```diff
@@ -22,6 +22,9 @@ Composite databases are managed using Cypher administrative commands.
 Therefore, you can create them as any other database in Neo4j standalone and cluster deployments without needing to deploy a dedicated proxy server (as with Fabric in Neo4j 4).
 For a detailed example of how to create a Composite database and add database aliases to it, see xref:composite-databases/administration.adoc[Managing Composite databases].
 
+Composite databases cannot guarantee compatibility between constituents in major releases. Therefore, it's recommended that all constituents belong to the same major release family.
```

Comment:
Maybe it would be good to define "constituents" before we have this text? In the introduction we seem to only mention "aliases". Wdyt?

Reply:
Good idea, updated